### DISCUSSION TITLE
Issue with OCR on a Specific Line Using PPOCRv4

### DISCUSSION THREAD

**Discussion by ep0p** [PaddlePaddle/PaddleOCR#14056] [Q&A]
I have a line in my document that isn't being recognized by the original PPOCRv4 model—it completely ignores it:
![wtf_line](https://github.com/user-attachments/assets/d500599f-89e1-4c55-9f75-ca6e5e891799)

When I use my recognition fine-tuned model, it detects the line but returns gibberish instead of the correct text.

Any ideas on why this is happening or how to fix it?

**Comment by Liyulingyue** [comment]
Perhaps your dictionary configuration is not suitable

**Reply by ep0p** [reply]
This issue isn't just happening with my fine-tuned model but also with the original PPOCRv4 model. When I run the inference using ```ocr = PaddleOCR(use_angle_cls=True, lang='fr')```, the line gets completely ignored.

After running some tests, I found that removing the trailing letters (as shown in the photo I shared) above this line allows the OCR to detect it properly. However, since this is part of a whole block of text, I’m unsure how to resolve it. Any suggestions on what to do in this case? Since i am sure this will not be the only case when this will happen.

**Reply by Liyulingyue** [reply]
Could you provide the full image(not only the crop part)? I wanna have a try.

**Reply by ep0p** [reply]
Unfortunately, I’m unable to share the full image as it contains confidential information. However, after further investigation, I re-exported the image using GIMP, and the OCR worked flawlessly on the new version.

It seems the issue was likely due to the original image's poor quality. 

That said, if you want to test it yourself by using the cropped portion of the image, you should be able to replicate the issue.

**Reply by Liyulingyue** [reply]
That's fine, I'm very glad your issue has been resolved.
